inspirehep/modules/theme/jinja2filters.py

```diff
@@ -756,3 +757,51 @@ def weblinks(description):
     if description:
         return 'Link to ' + description
     return 'Link to fulltext'
+
+
+@blueprint.app_template_filter()
+def jobs_similar(id):
+
+    out = ''
+
+    es_query = Query()
```

Comment:
If you rebase with latest `master` you will have to do this query with something like:

```python
x = RecordsSearch(index='records-jobs', doc_type='jobs')
x = x.query({
                        "more_like_this": {
                            "docs": [
                                {
                                    "_id": 123
                                }
                            ],
                            "min_term_freq": 0,
                            "min_doc_freq": 0,
                        }
                    })
x.execute()
```